app/gui/src/project-view/components/widgets/FileBrowserWidget/paths.ts

```diff
@@ -64,6 +65,11 @@ export function useFileBrowserStack(
   listDirectory: (dir: Directory) => Promise<readonly AnyAsset<AssetType>[]>,
 ) {
   const filenameInputContents = ref<string>('')
+  const fileExtensionInputContents = ref<string>('')
```

Comment:
The docs of this composable are not updated. Also, either change the name, or split to more composables with well-defined logics.

Reply:
Kaz did the refactoring.